build/build.sh

```diff
@@ -82,6 +82,17 @@ if [ "${BUILD_DOCX:-}" = "true" ]; then
     --defaults="$PANDOC_DEFAULTS_DIR/docx.yaml"
 fi
 
+# Spellcheck
+if [ "${SPELLCHECK:-}" = "true" ]; then
+  export ASPELL_CONF="add-extra-dicts $(pwd)/build/assets/custom-dictionary.txt; ignore-case true; ignore 1"
```

Comment:
```suggestion
  export ASPELL_CONF="add-extra-dicts $(pwd)/build/assets/custom-dictionary.txt;ignore-case true;ignore 1"
```

Reply:
maybe this is the issue and noting after the first parameter's actually getting used? 🤞